content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -42,6 +42,8 @@ gpgcheck=1
 repo_gpgcheck=1
 gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
 EOF
+rpm --import "https://packages.cloud.google.com/yum/doc/yum-key.gpg"
```

Comment:
I think this is wrong. You shouldn't use the `rpm` commands directly. If you use `yum`, the keys will get imported from the `gpgkey` location only if they are missing. @calebamiles PTAL